lib/util.py

```diff
@@ -201,11 +201,13 @@ def print_funct(*args, **kwargs):
 def conditional_raise(mask_exceptions):
```

Comment:
Should this function also forward previous exceptions?

Reply:
NA. This is re-raising the same/last exception (plain raise), not a new one. The masking function could handle and rise a new one, but the parameters are generic, it can be passed there if needed